community/kernel/src/main/java/org/neo4j/kernel/builtinprocs/ListIndexesProcedure.java

```diff
@@ -46,6 +49,7 @@ protected ListIndexesProcedure(ProcedureName procedureName)
         super( procedureSignature( procedureName )
                 .out( "description", Neo4jTypes.NTString )
                 .out( "state", Neo4jTypes.NTString )
+                .out( "unique", Neo4jTypes.NTBoolean )
```

Comment:
No, that's fair - forcing users to regex the text is no good either. WDYT about introducing `type` and `state` fields?

    CALL db.indexes()
    | description | type                 | state  |
    | INDEX ON .. | node_unique_property | ONLINE |
    | INDEX ON .. | node_label_property  | ONLINE |

That'd be future-proof as we add additional index types, and allows dependable programmatic introspection of state and type?